### DISCUSSION TITLE
Disabling vanilla villages

### DISCUSSION THREAD

**Discussion by MPCGomes** [ChoiceTheorem/ChoiceTheorem-s-overhauled-village#99] [Q&A] [answered]
Title :)

Edited by ChoiceTheorem: Is it possible to deactivate vanilla villages and use only CTOV's?

**Comment by ChoiceTheorem** [comment]
Can you give me your CTOV version?

**Reply by MPCGomes** [reply]
Of course! It's ctov-3.3.4 :)

**Comment by ChoiceTheorem** [comment, marked as answer]
If you use Fabric or Neoforge, update to 3.4! And use vanilla blocker
datapack or mod.

**Reply by MPCGomes** [reply]
Unfortunatelly, I'm on Minecraft 1.20.1 Forge =/

**Reply by ChoiceTheorem** [reply]
Update to 3.4.2!
